Adding Unshorten_URL_with_Slack workflow (#84)
rapid7/insightconnect-workflows: workflows/Unshorten_URL_with_Slack/help.md

# Description

> This workflow accepts a Slack command containing an URL, looks up the URL in unshorten.me API, and posts the full URL to slack

Reviewer: ```suggestion
This workflow accepts a Slack command containing a URL, looks up the URL with unshorten.me, and posts the full URL to Slack.
```

Sample Slack Trigger Commands:

`@Slack bot !unshorten https://bit.ly/39HoYMu`

# Key Features

* Lookup full URL from Slack

# Requirements

* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)

# Documentation

## Setup

Once the workflow has been imported, edit the workflow and setup or select your Slack connection in the _Unshorten URL Slack Trigger_ step.

To run the workflow, @ your Slackbot in the channel along with the command "!unshorten <URL>". The workflow will post a response in a thread.

## Technical Details

Plugins utilized by workflow:

|Plugin|Version|Count|
|----|----|--------|
|ExtractIt|2.0.0|1|
|Unshorten.me|1.0.3|1|

## Troubleshooting

> If a URL is not found to be shortened, the system will print "No Result Found"

Reviewer 2: Remove this, make the error message more explanatory.
